src/app/(proper_react)/(redesign)/terms/expiration-offer/View.tsx

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Link from "next/link";
+import Image from "next/image";
+import styles from "./View.module.scss";
+import monitorLogo from "../../../images/monitor-logo.webp";
+import { ExtendedReactLocalization } from "../../../../functions/l10n";
+
+export type Props = {
+  l10n: ExtendedReactLocalization;
+};
+
+export const View = (props: Props) => {
+  const l10n = props.l10n;
+
+  return (
+    <div className={styles.wrapper}>
+      <Link href="/" className={styles.logo}>
+        <Image
+          src={monitorLogo}
+          alt={l10n.getString("plus-expiration-header-logo-alt")}
+        />
+      </Link>
+      <div className={styles.terms}>
+        <h1>
+          {l10n.getFragment("plus-expiration-terms-heading", {
+            elems: { b: <b /> },
+          })}
+        </h1>
+        <ul>
+          <li>{l10n.getString("plus-expiration-terms-term1")}</li>
+          <li>{l10n.getString("plus-expiration-terms-term2")}</li>
+          <li>{l10n.getString("plus-expiration-terms-term3")}</li>
+          <li>{l10n.getString("plus-expiration-terms-term4")}</li>
+          <li>{l10n.getString("plus-expiration-terms-term5")}</li>
+          <li>
+            {l10n.getFragment("plus-expiration-terms-term6", {
+              elems: { "monitor-link": <Link href="/" /> },
```

Comment:
If we don’t know the `href` value, yet: Should we add a TODO comment?

Reply:
We do know the value; it's a link to the homepage (`/`) :)

![image](https://github.com/user-attachments/assets/e05fd7c7-7410-4bc2-929b-3d906356ace3)